docs/en/connector-v2/source/FtpFile.md

```diff
@@ -288,6 +294,39 @@ Source plugin common parameters, please refer to [Source Common Options](common-
 
 ```
 
+### Transfer Binary File
+
+```hocon
+
+env {
+  parallelism = 1
+  job.mode = "BATCH"
+}
+
+source {
+  FtpFile {
+    host = "192.168.31.48"
+    port = 21
+    user = tyrantlucifer
+    password = tianchao
+    path = "/seatunnel/read/binary/"
```

Comment:
**Issue**: The path provided in the configuration example uses a hard-coded value which might not be universally applicable or secure. <br> **Fix**: Replace the hard-coded path with a placeholder or a configuration parameter that can be set by the user or through environment variables. <br> **Code Suggestion**: 
 ```
 -    path = "/seatunnel/read/binary/"
 +    path = "${BINARY_FILES_PATH}"
 ```